src/main/java/uk/gov/hmcts/reform/prl/controllers/CallbackController.java

```diff
@@ -300,6 +301,8 @@ public AboutToStartOrSubmitCallbackResponse generateDocumentSubmitApplication(
             caseDataUpdated.putAll(documentGenService.generateDraftDocuments(authorisation, caseData));
         }
 
+        CaseDetails caseDetails = callbackRequest.getCaseDetails();
+        solicitorEmailService.sendAwaitingPaymentEmail(caseDetails);
```

Comment:
make one line call - solicitorEmailService.sendAwaitingPaymentEmail(callbackRequest.getCaseDetails());

Reply:
this will not work. make this call from FeeAndPayServiceRequestController when the event is submitted. not on about to submit.

Reply:
Yeah made this as a single line and also migrated this to the FeeAndPayServiceRequestController